jpype/_jcollection.py

```diff
@@ -54,6 +54,12 @@ def __delitem__(self, i):
         raise TypeError(
             "'%s' does not support item deletion, use remove() method" % type(self).__name__)
 
+    def __contains__(self, i):
+        try:
+            return self.contains(i)
```

Comment:
Why is it a TypError? I would have expected a KeyError, if i isn't contained.

Reply:
When we call a Java method and it can't be converted then it produces a TypeError.  We have to catch that TypeError and return a false, so that we can meet the Python contract for contains.